src/jetson/main.py

```diff
@@ -390,59 +87,59 @@ def drawFrame(boxes, frame, fps):
 
 if __name__ == "__main__":
     warnings.filterwarnings("once")
-    parser = argparse.ArgumentParser(description="Face detection")
-    parser.add_argument('--detector', '-d', type=str, required=True,
-                        help="Path to a trained face detector .pth file")
-    parser.add_argument('--detector_type', '-t', type=str, required=True, help="Type of face detector. One of "
-                                                                               "blazeface, ssd, or retinaface.")
-    parser.add_argument('--cuda', '-c', default=False,
-                        action='store_true', help="Enable cuda")
-    parser.add_argument('--classifier', type=str,
-                        help="Path to a trained classifier .pth file")
-    parser.add_argument('--write_imgs', default=False,
-                        help='Write images to output_dir')
-    parser.add_argument('--output_dir', default='encrypted_imgs',
-                        type=str, help="Where to output encrypted images")
-    args = parser.parse_args()
 
-    if args.detector_type not in DETECTOR_TYPES:
-        print(
-            'Please include a valid detector type (\'blazeface\', \'ssd\', or \'retinaface\'')
+    with open('src/jetson/config.json') as file:
+        args = json.load(file)
+
+    detector = args["DETECTOR"]
+    detector_type = args["DETECTOR_TYPE"]
+    cuda = args["CUDA"]
+    classifier = args["CLASSIFIER"]
+    send_to_database = args["SEND_TO_DATABASE"]
+    output_dir = args["OUTPUT_DIR"]
+    gstreamer = args["GSTREAMER"] # This should be true if running on jetson nano with picam
+    draw_frame = args["DRAW_FRAME"]
+
+    if detector_type not in DETECTOR_TYPES:
+        print('Please include a valid detector type (\'blazeface\', \'ssd\', or \'retinaface\'')
         exit(1)
 
     device = torch.device('cpu')
-    if args.cuda and torch.cuda.is_available():
+    if cuda and torch.cuda.is_available():
         device = torch.device('cuda:0')
 
-    g = torch.load(args.classifier, map_location=device)
+    g = torch.load(classifier, map_location=device)
```

Comment:
What is `g`? Goggle classifier?

Reply:
I did not write this code so I'm not sure what the original intention was with this variable name. We can change it to something more representative such as `torch_classifier` if that sounds fine

Reply:
I've changed the name to `classifier_model`